tests/migration/online_migration/post_migration.pm

```diff
@@ -24,7 +24,7 @@ sub run {
     select_console 'root-console';
 
     # print repos to screen and serial console after online migration
-    zypper_call('lr -u');
```

Comment:
Any reason why you need print the output there?

Reply:
Added description

Reply:
thanks